R/globals.R

```diff
@@ -25,6 +25,32 @@ register_s3_method <- function(pkg, generic, class, fun = NULL) {
   )
 }
 
+register_upgrade_message <- function(pkg, version) {
+  # Do nothing if the package isn't installed or
+  # meets the version requirement
+  if (system.file(package = pkg) == "") return(NULL)
+  if (packageVersion(pkg) >= as.character(version)) return(NULL)
```

Comment:
Instead of `as.character`, `package_version` would be more correct. But `Ops.numeric_version` will coerce the right had side to a numeric version object anyway, so I don't think there's a need to have either as.character` or `package_version`.

Reply:
Oh, interesting, thanks!